common/database.dart.func-sort-c.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
I don't think we should push coverage files into main. It doesn't make sense since coverage changes over time as we add tests but the HTML files are going to be static. If we do want to have coverage, we should make a badge on the README and have actions build a coverage report automatically